src/Ionide.KeepAChangelog/Library.fs

```diff
@@ -238,32 +226,60 @@ module Parser =
 
         attempt dmyDots <|> ymdDashes
 
+    let pYanked: Parser<_> =
+        opt (pstring "[YANKED]")
+        |>> function
+            | None -> false
+            | Some _ -> true
 
     let pVersion = mdUrl pSemver <|> pSemver
 
-    let pRelease: Parser<SemVersion.SemanticVersion * System.DateTime * ChangelogData option> =
+    let pReleaseDescription =
+        let nextHeader = (opt newline >>. regex @"[#]{1,3}\s\S" |>> ignore)
+        let endOfSection = choice [ eof; nextHeader ]
+
+        let description =
+            (manyTill anyChar (lookAhead endOfSection) .>> attempt (opt newline))
+            |>> function
+                | [] -> None
+                | x -> Some(System.String.Concat x)
+
+        description <?> "release description body"
+
+    let pRelease: Parser<Release> =
         let vPart = skipString "##" >>. spaces1 >>. pVersion
         let middle = spaces1 .>> pchar '-' .>> spaces1
-        let date = pDate .>> skipRestOfLine true
+        let date = pDate
+        let yanked = opt spaces1 >>. pYanked
```

Comment:
I was not able to keep the `.>> skipRestOfLine true` for some reason it was making the tests fails.